src/events/processTransfers/index.js

```diff
@@ -29,10 +29,15 @@ function processTransfersBuilder(config) {
       validatorContract = new web3Home.eth.Contract(bridgeValidatorsABI, validatorContractAddress)
     }
 
+    rootLogger.debug(`Processing ${transfers.length} Transfer events`)
     const callbacks = transfers.map(transfer =>
       limit(async () => {
         const { from, value } = transfer.returnValues
 
+        const logger = rootLogger.child({
```

Comment:
Is my understanding correct and we don't log any information about the transaction that raised this event below (similarly we have in `processAffirmationRequests`)? If it is true, let's add here a log message for this.

Reply:
Yes, you are right, thanks! Fixed in 10870a4a44ad6211f7b6c3a5c22392f9855a9416.